src/app/pages/details/common/hooks.js

```diff
@@ -35,6 +35,18 @@ function getUserStatusPromise() {
     };
 
     return userModel.load().then((user) => {
+        if (!user || Reflect.ownKeys(user).length === 0) {
```

Comment:
do we need these debug messages?

Reply:
Yes, there's an elusive bug that Brittany has run into. The debug messages will give some clues if she runs into it.